tests/unit/specs/components/LicenseDetailsCard.spec.js

```diff
@@ -0,0 +1,110 @@
+import { mount, createLocalVue, config } from '@vue/test-utils'
+import LicenseDetailsCard from '@/components/LicenseDetailsCard.vue'
+import { licenseSlug } from '@/utils/license-utilities'
+import VueI18n from 'vue-i18n'
+import Vuex from 'vuex'
+
+describe('LicenseDetailsCard.vue', () => {
+    let wrapper
+    let getters
+    let state
+    let store
+    let i18n
+
+    // Always creates a shallow instance of component
+    beforeEach(() => {
+            const localVue = createLocalVue()
+            localVue.use(VueI18n)
+            localVue.use(Vuex)
+            state = {
+                attributionDetails: {
+                    creatorName: 'J Doe',
+                    creatorProfileUrl: 'www.author.com',
+                    workTitle: 'My work',
+                    workUrl: 'www.author.com/pic.jpg'
+                }
+            }
+            getters = {
+                shortName: state => {
+                    return 'foo'
+                },
+                fullName: state => {
+                    return 'fooBar'
+                },
+                licenseUrl: state => (mode) => {
+                    return 'http://example.com'
+                },
+                iconsList: state => {
+                    return ['by', 'sa']
+                }
+            }
+            store = new Vuex.Store({
+                state,
+                getters
+            })
+            const messages = require('@/locales/en.json')
+            i18n = new VueI18n({
+                locale: 'en',
+                fallbackLocale: 'en',
+                messages: messages
+            })
+
+            config.mocks.i18n = i18n
+
+            config.mocks.i18n.$t = (key) => {
+                return i18n.messages[key]
+            }
+            wrapper = mount(LicenseDetailsCard, {
+                localVue,
+                store,
+                i18n
+            })
+        })
+        // Test for DOM elements which must be present
+    it('Has the main div tag', () => {
```

Comment:
These test names are very vague. Please make them more specific, stuff like "Main heading is present", or "Tabs are present".

Reply:
I'm sorry for keeping such names I have changes them. :)